aspnetcore/security/authentication/claims.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Mapping claims, customizing claims and transforming claims in ASP.NET Core
+author: damienbod
+description: Learn how to map claims, do claims transformations, customize claims.
+monikerRange: '>= aspnetcore-3.1'
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/29/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: security/authentication/claims
+---
+# Mapping claims, customizing claims and transforming claims in ASP.NET Core
+
+By [Damien Bowden](https://github.com/damienbod)
+
+Claims can be created from any user data or identity data which can be issued using a trusted identity provider or ASP.NET Core identity. A claim is a name value pair that represents what the subject is, not what the subject can do.
+This article covers the following areas:
+
+* How to configure and map claims using an OpenID Connect client
+* Set the name claim and the role claim
+* Reset the claims namespaces
+* Customize, extend the claims using IClaimsTransformation
```

Comment:
```suggestion
* Customize, extend the claims using <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation>
```

That's the interface, Or do you want the method: <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation.TransformAsync%2A>

Reply:
I customized using the IClaimsTransformation interface which has only one method, like you mentioned, so I'm not sure which is better

maybe this would be better, yes

Customize, extend the claims using <xref:Microsoft.AspNetCore.Authentication.IClaimsTransformation.TransformAsync>.

Reply:
Updated to use the TransformAsync as well